app/src/keymap.c

```diff
@@ -269,7 +269,8 @@ int zmk_keymap_sensor_triggered(uint8_t sensor_number, const struct device *sens
                 continue;
             }
 
-            ret = behavior_sensor_keymap_binding_triggered(binding, sensor, timestamp);
+            struct zmk_behavior_binding_event event = {.position = 0, .timestamp = timestamp};
```

Comment:
Thoughts on using `sensor_number` for the `position` value?

```suggestion
            struct zmk_behavior_binding_event event = {.position = sensor_number, .timestamp = timestamp};
```

Reply:
I was just thinking to add something like this earlier, we'd need something to differentiate between sensors